scene/gui/grid_container.cpp

```diff
@@ -256,15 +349,33 @@ int GridContainer::get_columns() const {
 	return columns;
 }
 
+void GridContainer::set_vertical(bool p_vertical) {
+	ERR_FAIL_COND_MSG(is_fixed, vformat("Can't change orientation of %s.", get_class()));
+	vertical = p_vertical;
```

Comment:
```suggestion
	if (vertical == p_vertical) {
		return;
	}
	vertical = p_vertical;
```
Looks like existing containers don't have this condition. We should add it eventually.